certified-connectors/Plumsail Documents/apiDefinition.swagger.json

```diff
@@ -260,24 +260,30 @@
             "description": "Internal server error, please write an email to support@plumsail.com"
           }
         },
-        "summary": "Triggers when a process is finished",
+        "summary": "Process finished",
```

Comment:
@RFlipper Summary value of triggers should start with "When... " or "Triggers...". Can you please revert it what it has before.

Reply:
@sriyen-msft Done, I changed it to `When process is finished`